README.md

```diff
@@ -1,371 +1,209 @@
 ```js
-function validateSelection (selection) {
-  // Code unchanged
-}
-
-function getPrice (selection) {
-  // Code unchanged
-}
-
-function vendingMachine (snackSelection, moneyInserted) {
-  if (validateSelection(snackSelection) === false) {
-    return 'Please select a valid snack.';
-  }
-
-  const price = getPrice(snackSelection);
-
-  if (price > moneyInserted) {
-    return `Please insert more to purchase ${snackSelection}.`;
-  }
-
-  console.log("In vendingMachine(), 'moneyInserted' contains:", moneyInserted);
-  console.log("In vendingMachine(), 'price' contains:", price);
-
-  const change = moneyInserted - price;
-
-  return `${snackSelection} dispensed. Your change is ${change}. Thank you!`;
-}
-```
-
-Now we get some additional information when running the code:
-```js
-vendingMachine('Ice cream', 100);
-// LOG: In vendingMachine(), 'moneyInserted' contains: 100
-// LOG: In vendingMachine(), 'price' contains: undefined
-// => "Ice cream dispensed. Your change is NaN. Thank you!"
-```
-
-So `moneyInserted` correctly contains the number `100`, but why the heck does `price` contain `undefined`? Well, looks like it gets assigned the return value from invoking the `getPrice()` function, so that function must be `return`ing `undefined`. We can use `console.log()` for more than simply checking the value of variables. This is a good opportunity to use it as a flag of whether or not the JavaScript engine reaches a particular line of code:
-```js
-function validateSelection (selection) {
-  // Code unchanged
-}
-
-function getPrice (selection) {
-  switch (selection) {
-    case 'Pretzels':
-      return 100;
-    case 'Chips':
-      return 75;
-    case 'Water':
-      return 50;
-  }
-
-  console.log("This logging statement will only run if none of the RETURN statements are hit.");
-}
-
-function vendingMachine (snackSelection, moneyInserted) {
-  // Code unchanged
-}
-
-vendingMachine('Ice cream', 100);
-// LOG: This logging statement will only run if none of the RETURN statements are hit.
-// LOG: In vendingMachine(), 'moneyInserted' contains: 100
-// LOG: In vendingMachine(), 'price' contains: undefined
-// => "Ice cream dispensed. Your change is NaN. Thank you!"
+console.log('Hello,', 'world!');
+// LOG: Hello, world!
 ```
 
-So none of the `return` statements in `getPrice()` are getting hit. Let's see what the `selection` variable contains:
-```js
-function validateSelection (selection) {
-  // Code unchanged
-}
-
-function getPrice (selection) {
-  console.log("In getPrice(), 'selection' contains:", selection);
-
-  switch (selection) {
-    case 'Pretzels':
-      return 100;
-    case 'Chips':
-      return 75;
-    case 'Water':
-      return 50;
-  }
-
-  console.log("This logging statement will only run if none of the RETURN statements are hit.");
-}
+Importantly, you can log not only simple things like `String`s or `Number`s but
+you can log objects and use disclosure triangles to "expand out" the contained
```

Comment:
disclosure triangles might not be self explanatory but I'm not sure how to get people to know what exactly we mean here. Maybe a "try clicking on one" example?

Reply:
We've mentioned them before, so I'm hoping they're safe. Google I'm feeling Lucky returns the right match for these two words, so let's see.